### PR TITLE
Terminus Release 3.1.0

### PR DESCRIPTION
**Pull request by rwagner00**
## Summary
**[Version Updates](https://pantheon.io/docs/terminus/updates)** - Terminus Release 3.1.0

## Effects
* https://github.com/pantheon-systems/terminus/pull/2399

## Release
- [x] When ready

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/partials/terminus-guide/eol.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ After this period, the version will reach End Of Life (**EOL**), and will no lon
 
 | Version           | Release Date       | EOL Date          |
 | ----------------- | ------------------ | ----------------- |
-| 3.0.9             | October 06, 2022   |                   |
+| 3.1.0             | October 18, 2022   |                   |
+| 3.0.9             | October 06, 2022   | October 18, 2032  |
```
Comment (kporras07) [inline source/partials/terminus-guide/eol.md:12]: *2023

**Review by michellecolon-pantheon — APPROVED** [review]
LGTM

**Comment by michellecolon-pantheon** [comment]
On hold: https://pantheon.slack.com/archives/C03SU7UCJ/p1666120112261859?thread_ts=1666117624.464359&cid=C03SU7UCJ

**Comment by michellecolon-pantheon** [comment]
It's a go for merging: https://pantheon.slack.com/archives/C02AJJM3KNH/p1666128893071419?thread_ts=1666050091.278519&cid=C02AJJM3KNH

**Pull request merged**